frontend/src/components/SeedlotRegistrationSteps/CollectionStep/index.tsx

```diff
@@ -0,0 +1,320 @@
+                invalidText={fieldsConfig.startDate.invalidText}
+                size="md"
+              />
+            </DatePicker>
+          </Column>
+          <Column sm={12} md={4} lg={6}>
+            <DatePicker
+              datePickerType="single"
+              dateFormat={DATE_FORMAT}
+              minDate={state.startDate}
+              value={state.endDate}
+              onChange={(_e: Array<Date>, selectedDate: string) => {
+                handleFormInput('endDate', selectedDate);
+              }}
+            >
+              <DatePickerInput
+                name={fieldsConfig.endDate.name}
+                placeholder={fieldsConfig.endDate.placeholder}
+                labelText={fieldsConfig.endDate.labelText}
+                helperText={fieldsConfig.endDate.helperText}
+                invalid={validationObj.isEndDateInvalid}
+                invalidText={fieldsConfig.endDate.invalidText}
+                size="md"
+              />
+            </DatePicker>
+          </Column>
+        </Row>
+        <Row>
+          <Column sm={12} md={4} lg={6}>
+            <NumberInput
+              name={fieldsConfig.numberOfContainers.name}
+              ref={numberOfContainersInputRef}
+              value={state.numberOfContainers}
+              label={fieldsConfig.numberOfContainers.labelText}
+              invalidText={fieldsConfig.numberOfContainers.invalidText}
+              onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
+                handleFormInput('numberOfContainers', e.target.value);
+              }}
+              hideSteppers
+              disableWheel
+            />
+          </Column>
+          <Column sm={12} md={4} lg={6}>
+            <NumberInput
+              name={fieldsConfig.volumePerContainers.name}
+              ref={volumePerContainersInputRef}
+              value={state.volumePerContainers}
+              label={fieldsConfig.volumePerContainers.labelText}
+              invalidText={fieldsConfig.volumePerContainers.invalidText}
+              onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
+                handleFormInput('volumePerContainers', e.target.value);
+              }}
+              hideSteppers
+              disableWheel
+            />
+          </Column>
+        </Row>
+        <Row>
+          <Column sm={12} md={12} lg={12}>
+            <NumberInput
+              name={fieldsConfig.volumeOfCones.name}
+              ref={volumeOfConesInputRef}
+              value={state.volumeOfCones}
+              label={fieldsConfig.volumeOfCones.labelText}
+              invalidText={fieldsConfig.volumeOfCones.invalidText}
+              helperText={fieldsConfig.volumeOfCones.helperText}
+              onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
+                handleFormInput('volumeOfCones', e.target.value);
+              }}
+              hideSteppers
+              disableWheel
+            />
```

Comment:
On figma, the default value of this field seems to be 1, not sure if that's correct, maybe align with the UX team? Also on figma, the value of this field should be calculated by the system.
![image](https://user-images.githubusercontent.com/105936322/231475204-4de859b6-ccbd-4a74-b0e6-18114ae63c91.png)

Reply:
I am checking with the team about the calculation and the warning message. I will fix it soon.

Reply:
@mgaseta Could you please review it again?